projects/plugins/jetpack/_inc/client/components/tree-selector/index.jsx

```diff
@@ -24,9 +25,8 @@ const TreeSelector = props => {
 			className="jp-tree-item"
 			style={ { marginLeft: isSearching ? 0 : item.depth * 25 } }
 		>
-			<input
-				type="checkbox"
-				id={ `jp-tree-item-${ item.id }` }
+			<CheckboxControl
+				id={ `jp-tree-item-${ item.ID }` }
```

Comment:
Maybe we should keep `id` instead `ID` here.

Reply:
Indeed, missed this one while rebasing. Thanks!